batch/niji_sample.ts

```
@@ -0,0 +1,184 @@
+import fs from "fs";
+import path from "path";
+
+const speakers = {};
+const beats = [];
+
+const nijivoice = [
+  { key: "hanamura_honoka", id: "231e0170-0ece-4155-be44-231423062f41" },
+  //  { key: "shitsuya_ren", id: "04c7f4e0-41d8-4d02-9cbe-bf79e635f5ab" },
+  { key: "fuyutsuki_hatsune", id: "d158278c-c4fa-461a-b271-468146ad51c9", name: "冬月 初音" },
+  ,
```

ESLint

[error] Unexpected comma in middle of array.

(no-sparse-arrays)